### ISSUE TITLE
D2 Weapon and Armor Speadsheets Don't Show Perks

### ISSUE THREAD

**Issue by tmuldrew** [DestinyItemManager/DIM#3021]
In Spreadsheets, the column labeled Perks is not showing any perks (probably needs additional columns), and there are no columns for other relevant data (Masterwork Level, Mods).

**Comment by glyphstalker** [comment]
I was gonna take a whack at fixing this, but I know CSV can be VERY particular if someone is expecting it formatted a specific way and it shows up a different way.

Are the columns currently defined set in stone, or would this be a good time to revise them to be more apt to the current state of the items being exported?

For instance, I recall reading a couple months ago that 'Light' was being replaced with 'Power', but that doesn't seem to have made it to the CSV output methods.  Also, it looks like Perks are much more significant now (from a tabular data perspective) than they were previously, so perhaps more than 1 column would be warranted?  Finally, would it make sense to change the column headers for Intellect, Strength and Discipline to Mobility, Resilience and Recovery when exporting D2 armor?

I'm sure there's more, but these are just my initial thoughts.

**Comment by kyleshay** [comment]
@glyphstalker most any changes for the d2 version are welcome -- so long as it still gets manually tested and works for the d1 version. 

I think that we should definitely update the columns to match the d2 verbiage

**Comment by glyphstalker** [comment]
Yeah, right after posting I pulled up my D1 character and noticed every column works as expected, so I think ideally I would just have it export a different schema for D2.
